validate-date.js

```diff
@@ -1,31 +1,98 @@
-function validateDate(dateValue) {
-    if (dateValue == null) {
-        return 'Invalid Format';
-    }
-    return dateValidator(dateValue);
+function validateDate(dateValue, responseType = "string", dateFormat = null) {
+  let responses = responseSetter(responseType)
+  if (dateValue == null) {
+    return responses[0];
+  }
+  return dateValidator(dateValue, responses, dateFormat);
+}
+
+function responseSetter(responseType) {
+  switch (responseType) {
+    case "string":
+      return ["Invalid Format", "Invalid Date", "Valid Date"];
+    case "boolean":
+      return [false, false, true];
+    default:
+      return ["Invalid Format", "Invalid Date", "Valid Date"];
+
+  }
 }
 
 function daysInMonth(year, month) {
-    const days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
-    return (month === 2 && year % 4 === 0) ? 29 : days[month - 1];
+  const days = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+  return month === 2 && year % 4 === 0 ? 29 : days[month - 1];
```

Comment:
Also, let's put back the "common braces ( )"? Although it has no significance technically I think it aids in readability. Thanks :)

Reply:
totally agree!

Reply:
Ok. So, usually with suggestions like this it is expected to be addressed right in this PR. Anyways, I will be cleaning up some stuff in the next PR and will fix this too.

Reply:
Oh! Great! I came to take a look on what I was doing wrong... I added those braces like 3 times hehe